### DISCUSSION TITLE
Apple broke login?

### DISCUSSION THREAD

**Discussion by QkSr4497** [Dadoum/iCode#4] [Q&A]
Hi,
It appears that apple broke the login API https://idmsa.apple.com/IDMSWebAuth/clientDAW.cgi

Was wondering if you maybe how to fix it or have an alternative?

thanks

**Comment by Dadoum** [comment]
Yes, I fixed it in another project ([Provision](https://github.com/Dadoum/Provision) !). Currently developing an alternative to Cydia Impactor using the new API for Linux.

**Comment by QkSr4497** [comment]
Your solution is to move to 2FA ? So you think App-Spesific is no longer
possible?

‫בתאריך יום ב׳, 18 ביולי 2022 ב-23:09 מאת ‪Dadoum‬‏ <‪
***@***.***‬‏>:‬

**Comment by Dadoum** [comment]
App specific password will unfortunately be an option only for paid developer accounts… but anyway a simple solution is to create a throwaway Apple account on your iPhone without any phone number and using a temp-mail to log-in.
